shared/fs/index.js

```diff
@@ -23,16 +25,20 @@ class Files extends React.PureComponent<FolderProps> {
   _renderRowPlaceholder = index => <Placeholder key={index} />
 
   render() {
-    const content =
-      this.props.progress === 'pending' ? (
-        <List items={['1', '2', '3']} renderItem={this._renderRowPlaceholder} />
-      ) : this.props.items.length === 0 ? (
-        <Box style={stylesEmptyContainer}>
-          <Text type="BodySmall">This is an empty folder.</Text>
-        </Box>
-      ) : (
-        <List items={this.props.items} renderItem={this._renderRow} />
-      )
+    const content = this.props.isUserReset ? (
+      <Box style={resetContainerStyle}>
+        <Icon type={isMobile ? 'icon-skull-64' : 'icon-skull-48'} />
+        <Icon type="icon-access-denied-266" />
+      </Box>
+    ) : this.props.progress === 'pending' ? (
+      <List items={['1', '2', '3']} renderItem={this._renderRowPlaceholder} />
+    ) : this.props.items.length === 0 ? (
+      <Box style={stylesEmptyContainer}>
+        <Text type="BodySmall">This is an empty folder.</Text>
+      </Box>
+    ) : (
+      <List items={this.props.items} renderItem={this._renderRow} />
+    )
```

Comment:
Ugh there's gonna be quite some conflicts here. I'll wait until yours is merged before merging mine.

Reply:
:(